test_scripts/Polices/Related_HMI_API/ATF_OnAllowSDLFunctionality_allowed_false_without_device.lua

```diff
@@ -0,0 +1,79 @@
+---------------------------------------------------------------------------------------------
+-- Description: 
+--     1. Preconditions: App is registered
+--     2. Steps: Activate App, send SDL.OnAllowSDLFunctionality with 'allowed=false' and without 'device' to HMI
+-- Requirement summary: 
+--     [Policy] "EXTERNAL_PROPRIETARY" flow: Related HMI API
+--     [Policies]: OnAllowSDLFunctionality with 'allowed=false' and without 'device' param from HMI
+--
+-- Expected result:
+--     In case PoliciesManager receives SDL.OnAllowSDLFunctionality with 'allowed=false' and without 'device' param from HMI, PoliciesManager must record 
+--     all of currently registered devices as NOT consented in Local PT ("device_data" - > "<device_id_1>", "<device_id_2>", etc. - >"user_consent_records"- > "device" sub-section).
+---------------------------------------------------------------------------------------------
+--[[ General Settings for configuration ]]
+Test = require('connecttest') 
+require('cardinalities')
+require('mobile_session')
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+--[[ Local Functions ]]
+local function get_id_value() 
+  local sql_select = "sqlite3 " .. tostring(SDLStoragePath) .. "policy.sqlite \"SELECT id FROM functional_group WHERE name = "BaseBeforeDataConsent"\""
+    local aHandle = assert( io.popen( sql_select , 'r'))
+    sql_output = aHandle:read( '*l' )   
+    local retvalue = tonumber(sql_output)    
+    if (retvalue == nil) then
+       self:FailTestCase("device id can't be read")
+    else 
+      return retvalue
+    end
+end
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+require('user_modules/AppTypes')
+--[[ General Settings for configuration ]]
+Test = require('connecttest') 
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeleteLogsFileAndPolicyTable()
```

Comment:
@vvvakulenko This function should be executed before SDL is started, because at this point you will clear db and logs that are used at the moment of SDL. 
The idea of this function is to have clear start of SDL - without logs, policy db....